pkg/db/postgres.go

```diff
@@ -0,0 +1,145 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+)
+
+// Pool is the global, exported database connection pool instance.
+// It will be nil if the connection failed.
+var Pool *pgxpool.Pool
+
+// InitPool attempts to initialize the PostgreSQL connection pool.
+// It logs a warning if initialization fails and sets Pool to nil, allowing the app to continue.
+func InitPool() {
+	connStr := os.Getenv("DATABASE_URL")
+	if connStr == "" {
+		// Log warning and return if the environment variable is not set
+		log.Println("WARNING: DATABASE_URL environment variable is not set. Database functions will be disabled.")
```

Comment:
Feels like this should be `log.warn("your mesage")` rather than embedding the word WARNING in the message itself. This applies to other logging statements in this file